Assets/MixedRealityToolkit.Tests/PlayModeTests/InteractableTests.cs

```diff
@@ -439,6 +439,61 @@ public IEnumerator TestDestroy()
             Assert.AreEqual(propBlock.GetColor("_Color"), FocusColor);
         }
 
+        [UnityTest]
+        /// <summary>
+        /// Tests button depth and focus state after enabling, disabling and re-enabling Interactable 
+        /// internally via SetDisable(true). The focus state after re-enabling should be false and button
+        /// depth should be in its default position.  This test is specifically addressing behavior described 
+        /// in issue 4967.
+        /// </summary>
+        public IEnumerator TestFocusStateOnDisableEnableInteractable()
+        {
+            var inputSimulationService = PlayModeTestUtilities.GetInputSimulationService();
+            int numSteps = 20;
+            Vector3 p1 = new Vector3(0.0f, 0f, 0f);
```

Comment:
nit: make Vector3.zero for p1 and p3, can you also fix in other tests that do this

Reply:
Yes I can. Vector3.zero is added to the tests that used it.